AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -1392,6 +1393,7 @@ quotev.com#$#body #res_area ::selection { background-color: #3297FD!important; }
 !
 !--- JS ----
 !
+toonnbook.nate.com#%#AG_onLoad(function(){ setTimeout(function() { Object.defineProperty(document.querySelector('div#wrap'), Object.keys(document.querySelector('div#wrap'))[1], {get: function() {}}); }, 15000); });
```

Comment:
This script will make a bug if time of website rendering is exceed than 15 seconds.
I am finding other solution.